os/board/rtl8730e/src/rtl8730e_mipi_lcdc.c

```diff
@@ -280,6 +281,8 @@ u32 rtl8730e_hv_isr(void *Data)
 			lcddbg("ERROR: LCDC DMA Underflow-----\n");
 			InterruptRegister((IRQ_FUN)rtl8730e_mipidsi_underflowreset, mipi_irq_info.num, (u32)mipi_irq_info.data, mipi_irq_info.priority);
 			InterruptEn(mipi_irq_info.num, mipi_irq_info.priority);
+			g_rtl8730e_config_dev_s.underflow = 1;
```

Comment:
I think, we need to clear the LCDC state and interrupt or waiting done lcdc when priv->mipi_mode_switch(CMD_MODE).

Reply:
how about disable underflow irq when cmd mode? 
LCDC_INTConfig(pLCDC, LCDC_BIT_DMA_UN_INTEN , DISABLE);

Reply:
Okay it looks goot.

And SMP case, we can get a raice condition issue about `g_rtl8730e_config_dev_s.underflow` .
Core 0 - already in the LCDC IRQ.
Core 1 - disable usderflow irq -> mipi_transfer -> check `g_rtl8730e_config_dev_s.underflow`

How about the protect `g_rtl8730e_config_dev_s.underflow` using enter_critical_section()?
(it has spin_lock)

Reply:
noted, i will add the protect when i get back office ;)

Reply:
Hi @ewoodev ,
Could you review the new commit again? I've added spin lock around mipi transfer and isr that modify g_rtl8730e_config_dev_s.underflow.